packages/playground/src/components/index.ts

```diff
@@ -0,0 +1,2 @@
+// @ts-ignore
```

Comment:
Playground is a .jsx file and TS doesn't like it. It should be removed after rewrite

Reply:
Ah... I missed that. When do you plan to rewrite it as .tsx?

Reply:
I'm working on it. I'm not experienced with class components this complicated. There are some workarounds in the code that I don't quite understand the reason behind them. For example:

```js
// force resetting form component instance
    this.setState({ form: false }, () =>
      this.setState({
        ...data,
        form: true,
        ArrayFieldTemplate,
        ObjectFieldTemplate,
        uiSchema,
        extraErrors,
      })
    );
```

What's the reason for setting the form to false and then true? If it's to force resetting some state using effects then we need some redesigning.

Also the variable name "form" as a boolean doesn't really make sense to me.

Reply:
If you renamed it to `showForm` then it would probably make a little bit more sense. Essentially it seems like there may have been issues with showing the form while reloading some of the properties, so they hide it during the transition.

Reply:
> If you renamed it to `showForm` then it would probably make a little bit more sense.

Good idea

> Essentially it seems like there may have been issues with showing the form while loading in new properties from the switching of examples, so they hide it during the transition. Or maybe it just looked ugly during the transition?

Seems like we'll need to rethink some parts. I just encountered this comment in DemoFrame: `// TODO: find a better way to render fluent-ui in an iframe, if we need to do so.`